packages/rancher-cis-benchmark/charts/templates/configmap.yaml

```diff
@@ -13,4 +13,5 @@ data:
     >=1.20.5: rke2-cis-1.6-profile-permissive
   eks: "eks-profile"
   gke: "gke-profile"
+  k3s: "k3s-cis-1.6-profile-permissive"
```

Comment:
is there any variation based on k8s versions? Or for all versions "k3s-cis-1.6-profile-permissive" is the default?

Reply:
`k3s-cis-1.6-profile-permissive` will be the default.